Tiltfile

```diff
@@ -464,25 +470,61 @@ docker_build(
     ],
 )
 
-if spy_relayer:
+if spy_relayer or redis or generic_relayer or ci_tests:
     docker_build(
         ref = "redis",
         context = ".",
         only = ["./third_party"],
         dockerfile = "third_party/redis/Dockerfile",
     )
 
-    k8s_yaml_with_ns("devnet/redis.yaml")
-
+if spy_relayer or redis or ci_tests:
     k8s_resource(
         "redis",
         port_forwards = [
             port_forward(6379, name = "Redis Default [:6379]", host = webHost),
         ],
-        labels = ["spy-relayer"],
+        labels = ["redis"],
+        trigger_mode = trigger_mode,
+    )
+
+    k8s_yaml_with_ns("devnet/redis.yaml")
+
+if generic_relayer or ci_tests:
+    k8s_resource(
+        "redis-relayer",
+        port_forwards = [
+            port_forward(6378, name = "Generic Relayer Redis [:6378]", host = webHost),
+        ],
+        labels = ["redis-relayer"],
         trigger_mode = trigger_mode,
     )
 
+    k8s_yaml_with_ns("devnet/redis-relayer.yaml")
+
+
+
+if generic_relayer:
+    k8s_resource(
+        "relayer-engine",
+        resource_deps = ["guardian", "redis-relayer", "spy"],
+        port_forwards = [
+            port_forward(3003, container_port=3000, name = "Bullmq UI [:3003]", host = webHost),
+        ],
+        labels = ["relayer-engine"],
+        trigger_mode = trigger_mode,
+    )
+    docker_build(
+        ref = "relayer-engine",
+        context = ".",
+        only = ["./ethereum", "./relayer/generic_relayer", "./sdk", "./solana"],
```

Comment:
The ethereum folder, sdk folder, and solana folders all can be removed from here after the change of importing the SDK is made. It is currently needed because the sdk depends on /ethereum and /solana. But, once the sdk is published, this can be imported easily without all these additional dependencies.